shared/reducers/pinentry.js

```diff
@@ -39,6 +40,17 @@ const initialState: RootPinentryState = {
 export default function (state: RootPinentryState = initialState, action: PinentryActions): RootPinentryState {
   const sessionID: ?number = (action.payload && action.payload.sessionID != null) ? action.payload.sessionID : null
   switch (action.type) {
+    case CommonConstants.resetStore:
+      if (state.started === 1) {
```

Comment:
The type of this is:

```
 export type RootPinentryState = {
   started: 1,
   pinentryStates: {
     [key: number]: PinentryState
   }
 } | {
   started: 0
 }
```

I think you're suggesting:

```js
return {...initialState, started: state.started}
```

but that will fail since depending on whether started is 1 or 0 (this was before flow allowed disjoints on booleans) we have or don't have `pinentryStates`.

Since initialState has no `pinentryStates` you enter an invalid type with `{started: 1}`